libvtcapture/src/vtcapture_openapi.c

```diff
@@ -0,0 +1,7 @@
+#include "vtcapture/vtcapture_openapi.h"
```

Comment:
Could be named `vtCaptureApi_c.c`. While in system library it's actually a C++ source file, using C is better IMO.

Reply:
I've renamed it.